Tests/SystemTests/Utils/URLSessionServerTrustSync.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  File.swift
+//  
+//
+//  Created by Łukasz Szyszkowski on 16/08/2021.
+//
+
+import Foundation
+import XCTest
+
+class URLSessionServerTrustSync: NSObject, URLSessionDelegate, URLSessionTaskDelegate {
```

Comment:
Could you please add some comments about what this does. The name is not so friendly. Is the class being used anywhere, I can't find the usage?

Reply:
For example, with documentation I would know that the sync is for **synchronous**, not **synchronize**. These are utility methods for being able to return values instead of using callbacks. 

I still can't find the usage of them though.

Reply:
Ok, I'll add comments, this is a copy of helper from `ably-cocoa`

Reply:
1565f2057cdf178e6e1bc1ab61dc90a3b06520b6

Reply:
Can you show me where this class is being used? I think you might have missed when I said this at first

> Is the class being used anywhere, I can't find the usage?

Reply:
Okay, got it, its in `JWTHelper.swift`, 
```
let (responseData, responseError, _) = URLSessionServerTrustSync().get(request)
```